### PR TITLE
spotify fix

### PR DESCRIPTION
**Pull request by Baturax**
- I tested the changes in this PR: YES

- This new package conforms to the package-requirements: YES

- I built this PR locally for my native architecture, (ARCH-LIBC)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - x86_64 glibc

### CODE REVIEW

File: srcpkgs/spotify/template [hunk 1]
```diff
@@ -11,8 +11,8 @@ short_desc="Proprietary music streaming client"
 maintainer="Stefan Mühlinghaus <jazzman@alphabreed.com>"
 license="custom:Proprietary"
 homepage="https://www.spotify.com"
-distfiles="http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_${version}.${_subver}_amd64.deb"
```
Comment (classabbyamp) [inline srcpkgs/spotify/template:14]: don't hardcode, use the variables above

**Review by classabbyamp — CHANGES_REQUESTED** [review]

File: srcpkgs/spotify/template (hunk 1 above)
Comment (Baturax) [reply srcpkgs/spotify/template:14]: it gives error 
http://repository.spotify.com/pool/non-free/s/spotify-client/spotify-client_1.2.26.1187.g36b715a1_amd64.deb: Not Found
=> ERROR: spotify-1.2.26_1: failed to fetch 'spotify-client_1.2.26.1187.g36b715a1_amd64.deb'.

Comment (classabbyamp) [reply srcpkgs/spotify/template:14]: update the variables instead of removing them and replacing them with new values.

Comment (Baturax) [reply srcpkgs/spotify/template:14]: fixed thanks

**Comment by dataCobra** [comment]
Hey, there is already a PR for an update to the newest version where we also discussed the no longer available version 1.2.26.

PR -> https://github.com/void-linux/void-packages/pull/48955

**Comment by Baturax** [comment]
> Hey, there is already a PR for an update to the newest version where we also discussed the no longer available version 1.2.26.
> 
> PR -> #48955

oh okay. will you accept it?

**Comment by classabbyamp** [comment]
#49559

**Pull request closed without merging**